CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 - DNA Tester works now with more than one fingerprint on a weapon
 - TraitorButton config files should now actually work
 - Translation strings not rendering on detective's body search mode combobox
+- C4 defusal prompt suggesting the wrong key
```

Comment:
We should do our changelog in the same way.
Either we say:
Fixed:
-"Whats wasnt working"
or
-"How it should now behave correctly"

Here the first one is used.
In my opinion the second should be better.
Therefore I'd suggest changing it to
```suggestion
- C4 defusal prompt now suggesting the right key
```

Reply:
Since added and changed show the new state, this should probably be handled the same

Reply:
having this be consistent would be nice, but i don't care too much about this. From context it is pretty clear what was done here:

* The old version would read: 'Fixed C4 defusal prompt suggesting the wrong key'
* your version would read: 'As of this change (the) C4 defusal prompt (is) now suggesting the right key'

Reply:
However i do like Tim's approach. Simple and to the point.